### PR TITLE
Fix #268: Failing GA for Docker build

### LINKED ISSUE #268: Failing GA for Docker build

### PR DESCRIPTION
**Pull request by jandusil**
- Add copy lb step to GA docker build

### CODE REVIEW

**Comment by jandusil** [comment]
TESTED after finally getting all the paths correctly https://github.com/wultra/powerauth-tests/actions/runs/6697172529

File: .github/workflows/develop_pa-test-internal-testserver-app.yml [hunk 1]
```diff
@@ -49,6 +49,8 @@ jobs:
         registry: https://powerauthextendedtest.azurecr.io/
         username: ${{ secrets.AZUREAPPSERVICE_CONTAINERUSERNAME }}
         password: ${{ secrets.AZUREAPPSERVICE_CONTAINERPASSWORD }}
+    - name: Copy liquibase files
+      run: powerauth-test-server/copy_liquibase.sh
```
Comment (zcgandcomp) [inline .github/workflows/develop_pa-test-internal-testserver-app.yml:53]: The script is usually expected to be executed in the context of directory he resides in. I suggest to do
```
run: |
        cd powerauth-test-server
       ./copy_liquibase.sh
```
and leave the copy_liquibase.sh script unchanged.

**Review by zcgandcomp — CHANGES_REQUESTED** [review]

File: .github/workflows/develop_pa-test-internal-testserver-app.yml (hunk 1 above)
Comment (jandusil) [reply .github/workflows/develop_pa-test-internal-testserver-app.yml:53]: Thanks for the tip!

**Comment by jandusil** [comment]
https://github.com/wultra/powerauth-tests/actions/runs/6716252652

**Review by zcgandcomp — APPROVED** [review]
Looks OK.

**Pull request merged**